gradle-spaghetti-typescript-plugin/src/main/java/com/prezi/spaghetti/typescript/gradle/internal/MergeDtsTask.java

```diff
@@ -31,7 +26,7 @@
     private File sourceDir;
     private File tsCompilerPath = null;
 
-    @Input
+    @Internal
     public File getWorkDir() {
         return workDir;
     }
```

Comment:
I wonder why we never used `Task.getTemporaryDir()` for this instead. I guess we didn't know it existed. :) In any case, I guess it's good to have it now. We'll probably have something like `@TemporaryDirectory` in Gradle to mark these properties in the future.